patches/Lutris/metainfo-license.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net.lutris.Lutris.metainfo.xml	2020-11-15 01:09:28.542949609 +0100
++++ b/net.lutris.Lutris.metainfo.xml	2020-11-15 01:09:42.637072978 +0100
+@@ -3,7 +3,7 @@
+ <component type="desktop">
+   <id>net.lutris.Lutris</id>
+   <project_license>GPL-3.0-or-later</project_license>
+-  <metadata_license>GPL-3.0-or-later</metadata_license>
++  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
I don't think you can just change license of an upstream file, especially if it's GPL.

Reply:
Yes, I contacted Lutris team for this precise reason, for the moment I just want to see if it builds... I need to fix the patch